osd_install_access_delete_cluster/creating-a-gcp-cluster-with-workload-identity-federation.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="osd-creating-a-cluster-on-gcp-with-workload-identity-federation"]
+= Creating a cluster on GCP with Workload Identity Federation
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: osd-creating-a-cluster-on-gcp-with-workload-identity-federation
+
+toc::[]
+
+include::modules/wif-overview.adoc[leveloffset=+1]
+
+[id="osd-creating-a-cluster-on-gcp-prerequisites1_{context}"]
+== Prerequisites
+
+* You have confirmed your Google Cloud account has the necessary resource quotas and limits to support your desired cluster size according to the cluster resource requirements.
```

Comment:
@mletalie I know we discussed this previously, but I'm not sure here is the best place for the prereqs as they are separated from the creating a cluster with the CLI procedure. I think they should both come in under the two procedures (OCM and CLI)  If you have decided to move them since we last spoke, then you can just ignore this comment.